### DISCUSSION TITLE
DTS 1.3 Provider

### DISCUSSION THREAD

**Discussion by pals-ucb** [openssl/openssl#21847] [Q&A]
Hello,
    I am a student in UCB on the MICS program. Few of us are researching on PQC on IoT Devices and we were using DTLS 1.2 and openSSL1.1 along with OQS. Now, with the deprecation of 1.1, we want to move to 3.0.  We realized that OpenSSL 3.0 requires development of DTL1.3 provider. After researching and asking around, we figured that the best way for us to approach is to develop DTL1.3 provider for openSSL3.0 or join someone who is working on that. 
     I am wondering if anyone is working on DTLS 1.3 provider, if so who/links?.   Otherwise, can we get started on this effort. I am thinking of taking an experimental fork and working on that. This will help anyone else interested to join.

/pals

**Comment by mattcaswell** [comment]
See #13900 for the latest status. Note that implementation of DTLS1.3 won't really be via a provider since there are no provider hooks for implementing new protocol versions.
